### PR TITLE
initial commit of new about view

### PR DESCRIPTION
**Pull request by braathen**

### CODE REVIEW

File: qml/AboutView.qml [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    Keys.onEscapePressed: navigator.home()
+
+    Accessible.ignored: true
+
+    property string title: ""
+
+    ColumnLayout {
+        id: content
+        Layout.alignment: Qt.AlignHCenter | Qt.AlignTop
+//        height: deviceInfo.implicitHeight + deviceConfig.implicitHeight
```
Comment (dagheyman) [inline qml/AboutView.qml:37]: Drop comment

File: qml/AboutView.qml [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+                    opacity: highEmphasis
+                    Layout.alignment: Qt.AlignLeft | Qt.AlignVCenter
+                    Layout.leftMargin: 8
+                }
+            }
+
+            StyledExpansionContainer {
+                Layout.topMargin: 24
+
+                StyledExpansionPanel {
```
Comment (dagheyman) [inline qml/AboutView.qml:71]: Change to support link, and use Yubico Authenticator as the name

File: qml/AboutView.qml [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+                    }
+                }
+            }
+
+            ColumnLayout {
+                Layout.leftMargin: 16
+                Layout.topMargin: 24
+
+                Label {
+                    text: qsTr("Yubico Authenticator (v5.1.0)")
```
Comment (dagheyman) [inline qml/AboutView.qml:107]: Get version from the variable, as before

File: qml/NavigationDrawer.qml [hunk 4]
```diff
@@ -115,15 +115,9 @@ Drawer {
...
-            text: "About"
-            onActivated: navigator.confirm({
-                       "message": qsTr("Yubico Authenticator v%1").arg(appVersion),
-                       "description": qsTr("Copyright © " + Qt.formatDateTime(new Date(),"yyyy") + ", Yubico AB." +
-                                           qsTr("\n\nAll rights reserved.")),
-                       "warning": false,
-                       "buttons": false,
-                       "noicon": true
-                   })
+            text: "About/Help"
```
Comment (dagheyman) [inline qml/NavigationDrawer.qml:118]: Change to About

**Comment by fdennis** [comment]
It looks like the logo in about is quite blurry... Should we change this?

**Pull request merged**